include/sdbus-c++/Error.h

```diff
@@ -43,6 +43,11 @@ namespace sdbus {
         : public std::runtime_error
     {
     public:
+        explicit Error(const std::string& name, const char* message = nullptr)
+            : Error(name, std::string(message ? message : ""))
```

Comment:
Message parameter with default nullptr value is exactly what I thought about when I saw your commit in your repo. Glad that you've added it (and the tests too)